MatrixKit/Views/ReadReceipts/MXKReadReceiptTableViewCell.m

```diff
@@ -0,0 +1,41 @@
+/*
+ Copyright 2017 Aram Sargsyan
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MXKReadReceiptTableViewCell.h"
+
+
+@implementation MXKReadReceiptTableViewCell
+
+- (void)awakeFromNib
+{
+    [super awakeFromNib];
+    [self configureAvatarImageView];
+}
+
+- (void)configureAvatarImageView
+{
+    self.avatarImageView.layer.cornerRadius = CGRectGetWidth(self.avatarImageView.frame)/2;
```

Comment:
You have to override `- (void)layoutSubviews` to set correctly the corner radius. Set this value during `awakeFromNib` may be too early.

Please use the following code:
`- (void)layoutSubviews
{
    [super layoutSubviews];
    
    if (self.avatarImageView)
    {
        // Round image view
        [self.avatarImageView.layer setCornerRadius:self.pictureView.frame.size.width / 2];
        self.avatarImageView.clipsToBounds = YES;
    }
}`

You should move `self.avatarImageView.enableInMemoryCache = YES;` in `awakeFromNib`